source/braille.py

```diff
@@ -1553,7 +1552,7 @@ class BrailleHandler(baseObject.AutoPropertyObject):
 
 	def __init__(self):
 		louisHelper.initialize()
-		self.display = None
+		self.display = None  # BrailleDisplayDriver

```

Comment:
It looks like something like this is allowed:

```suggestion
		self.display: BrailleDisplayDriver = None 
```